scripts/us_eia/opendata/process/schema.mcf

```diff
@@ -0,0 +1,63 @@
+# New units
+
+Node: dcid:BtuPerPound
+typeOf: dcs:EnergyUnitOfMeasure
+description: "Energy, in BTU's, per pound of fuel."
+name: "BtuPerPound"
+
+Node: dcid:ShortTon
```

Comment:
Interesting!  I think in the Electricity dataset too I should use ShortTon instead of MetricTon.

From https://www.eia.gov/opendata/qb.php?category=446&sdid=ELEC.CONS_EG.PC-US-99.M its not clear whether its Metric or Short, because the glossary refers to both as Ton (https://www.eia.gov/energyexplained/units-and-calculators/), instead of Tonne (what wikipedia article suggests).

But matching up the Petroleum Coke numbers from https://www.eia.gov/totalenergy/data/monthly/pdf/sec7_9.pdf, it does seem to be ShortTon too... I'll make the change in a follow up PR.

Reply:
wow, that's a nice find